viewing_party/main.py

```diff
@@ -0,0 +1,147 @@
+# WAVE 1
+
+def create_movie(title, genre, rating):
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    for value in movie.values():
+        if value is None:
+            return None
+    return movie    
```

Comment:
this works! one thing to consider is that a new variables is made, stored in memory, then a for loop runs before it checks if any of the parameters are valid, maybe consider:

```suggestion
if title and genre and rating:
    movie = {
        "title": title,
        "genre": genre,
        "rating": rating
    }
        return movie
    else:
        return None
```

now the variable won't be made and take up memory space unless the parameters are valid